spec/index.bs

```diff
@@ -1803,6 +1803,7 @@ dictionary GPUSamplerDescriptor : GPUObjectDescriptorBase {
 - If {{GPUSamplerDescriptor/compare}} is provided, the sampler will be a comparison sampler with the specified
     {{GPUCompareFunction}}.
 - {{GPUSamplerDescriptor/maxAnisotropy}} specifies the maximum anisotropy value clamp used by the sampler.
+    Most implementations support values in range between 1 and 16, inclusive.
```

Comment:
Should be `Note:`?

Reply:
I don't know how to put a note into this block. Moved outside now.